getFeatureFlagById/main.go

```diff
@@ -6,37 +6,71 @@ import (
 	"net/http"
 
 	"github.com/Real-Dev-Squad/feature-flag-backend/database"
+	"github.com/Real-Dev-Squad/feature-flag-backend/jwt"
+	"github.com/Real-Dev-Squad/feature-flag-backend/middleware"
 	"github.com/Real-Dev-Squad/feature-flag-backend/utils"
 	"github.com/aws/aws-lambda-go/events"
 	"github.com/aws/aws-lambda-go/lambda"
 )
 
 func handler(req events.APIGatewayProxyRequest) (events.APIGatewayProxyResponse, error) {
-	id, _ := req.PathParameters["flagId"]
+	corsResponse, err := middleware.CORSMiddleware()(req)
+	if err != nil {
+		log.Printf("CORS error: %v", err)
+		return corsResponse, err
+	}
+
+	if corsResponse.StatusCode != http.StatusOK {
+		return corsResponse, nil
+	}
+
+	response, userId, err := jwt.JWTMiddleware()(req)
+	if err != nil {
+		log.Printf("JWT middleware error: %v", err)
+		return response, err
+	}
+
+	if response.StatusCode != http.StatusOK {
+		return response, nil
+	}
+
+	log.Printf("User ID: %s", userId)
+
+	id, ok := req.PathParameters["flagId"]
+	if !ok {
+		log.Println("flagId is required")
```

Comment:
- let's make this more descriptive, making it easy to identify the origin of this error log on a server. 
```suggestion
		log.Println("ERROR: getFeatureFlagById : Flag id is missing")
```

Reply:
the logs are printed seperately for every lambda